worker/pkg/workflows/datasync/workflow/workflow.go

```diff
@@ -420,7 +421,7 @@ func runRedisCleanUpActivity(
 					},
 					HeartbeatTimeout: 1 * time.Minute,
 				}),
-				syncrediscleanup_activity.DeleteRedisHash,
+				redisCleanUpActivity,
```

Comment:
Missing the `.DeleteRedisHash`

Slightly concerning the tests didn't catch this, but maybe it suffers the same fate as the post table sync.